app/models/concerns/search/sources.rb

```diff
@@ -0,0 +1,19 @@
+module Search
```

Comment:
Don't think this is needed any longer now that the other changes to the source model has been removed